samples/dnn/action_recognition.py

```diff
@@ -0,0 +1,280 @@
+import os
+import copy
+import shutil
+import numpy as np
+import cv2 as cv
+import argparse
+
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--input', help='Path to input video file.')
+parser.add_argument('--model', help='Path to model.')
+parser.add_argument('--classes', help='Path to classes list.')
+parser.add_argument('--temporal_unit', default=5, help='Averages the scores over temporal_unit x sample_duration clips.')
+parser.add_argument('--sample_duration', default=16, help='Number of frames for action detection.')
```

Comment:
Can it be different from 16? If not, please define as a constant (i.e. SAMPLE_DURATION=16)